src/main/scala/it/pagopa/interop/authorizationprocess/api/impl/ClientApiServiceImpl.scala

```diff
@@ -708,4 +651,22 @@ final case class ClientApiServiceImpl(
     }
   }
 
+  override def getClients(name: Option[String], relationshipIds: String, offset: Int, limit: Int)(implicit
+    contexts: Seq[(String, String)],
+    toEntityMarshallerClients: ToEntityMarshaller[Clients],
+    toEntityMarshallerProblem: ToEntityMarshaller[Problem]
+  ): Route = authorize(ADMIN_ROLE, SECURITY_ROLE, M2M_ROLE) {
+    val operationLabel =
+      s"Retrieving clients by name $name , relationship $relationshipIds"
+    logger.info(operationLabel)
+
+    val result: Future[Clients] = for {
```

Comment:
⚠️ the [permissions check](https://github.com/pagopa/interop-be-authorization-process/pull/203/files?diff=split&w=1#diff-209aa1b53c41d816589a5ccda66c676d1eb0c523ee48138033a3e7e3d4e27b85L159) has been removed. 
It is required to 
- allow admin users to see all their clients, even if they are not included in the relationships
- if user is not an admin, return only clients that includes the relationship id of the user-entity